plugins/ccgx/fu-ccgx-dmc-device.c

```diff
@@ -0,0 +1,703 @@
+/*
+ * Copyright (C) 2020 Cypress Semiconductor Corporation.
+ * Copyright (C) 2020 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include "fu-chunk.h"
+
+#include "fu-ccgx-common.h"
+#include "fu-ccgx-dmc-common.h"
+#include "fu-ccgx-dmc-device.h"
+#include "fu-ccgx-dmc-firmware.h"
+
+#define DMC_FW_WRITE_STATUS_RETRY_DELAY_MS 30 /* 30 msec */
+
+struct _FuCcgxDmcDevice {
+	FuUsbDevice		parent_instance;
+	FWImageType		fw_image_type;
+        DmcDockIdentity		dock_id;
+        guint8			ep_intr_in;
+        guint8			ep_bulk_out;
```

Comment:
tabs

Reply:
Fixed it on today commit (ccgx: fix code alignment, modify code for review)